src/lang/builtins_json.ml

```diff
@@ -47,6 +47,7 @@ let rec to_json_compact v =
           Printf.sprintf "[%s]" (String.concat "," (List.map to_json_compact l))
         )
     | Lang.Tuple l -> "[" ^ String.concat "," (List.map to_json_compact l) ^ "]"
+    | Lang.Meth _ -> failwith "TODO: JSON of field not yet implemented"
```

Comment:
I think it could be okay to do `"<field>"` or something like that here. That's what's done with requests and other abstract types. Also, let's update the doc accordingly.